test/system/600-completion.bats

```diff
@@ -309,6 +309,12 @@ function _check_no_suggestions() {
     # recurse for any subcommands.
     check_shell_completion
 
+    # check inspect with format flag
+    run_podman create --name inspect-$random_container_name $IMAGE
```

Comment:
no need to create another container, lets just reuse `created-$random_container_name`